.travis.yml

```diff
@@ -23,6 +23,10 @@ matrix:
     - php: nightly
     - env: SYMFONY_VERSION=3.1.*@dev
 
-install: composer update ${COMPOSER_OPTIONS}
+install:
+  - composer global require sllh/composer-lint:@stable --prefer-dist --no-interaction
```

Comment:
No need, we already are on `install` section.

This will add two commands for basically the same thing.

This how this is done for sonata: https://github.com/sonata-project/dev-kit/blob/4d1fa959f655fca40f998c0e3a994986ab79def2/project/.travis/install_lint.sh#L4